### PR TITLE
feat : add css style to simple Table container - EXO-61139 - Meeds-io/MIPs#37

### PR DESCRIPTION
**Pull request by sofyenne**
we will add the css style to newly added container named SimpleTableContainer.

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/platform.less [hunk 1]
```diff
@@ -19,6 +19,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 @import "Sprites/Style.less";
 @import "uiToolbarContainer/Style.less";
 @import "perfect-scrollbar.less";
+@import "uiSimpleContainer/simpleTableContainer.less";
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/platform/skin/platform.less]:
```suggestion
@import "uiSimpleContainers/simpleTableContainer.less";
```

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainer/simpleTableContainer.less [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/******* SimpleTableContainer *******************/
+
+.UIContainer.SimpleTableContainer { //normal view container
+    margin-top: 20px;
+  }
+  
+.UIContainer.SimpleTableContainer.EdittingContainer { //edit mode container 
+    margin-top:0px
+  }
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainer/simpleTableContainer.less]:
```suggestion
.UIContainer.SimpleTableContainer { //normal view container
  margin-top: 20px;
}
  
.UIContainer.SimpleTableContainer.EdittingContainer { //edit mode container 
  margin-top:0px
}
```

**Review by azayati — CHANGES_REQUESTED** [review]

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainer/simpleTableContainer.less [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/******* SimpleTableContainer *******************/
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainer/simpleTableContainer.less]: to be removed

**Review by azayati — CHANGES_REQUESTED** [review]

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/platform.less (hunk 1 above)
Comment (sofyenne) [reply platform-ui-skin/src/main/webapp/skin/less/platform/skin/platform.less]: done !

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainer/simpleTableContainer.less (hunk 2 above)
Comment (sofyenne) [reply platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainer/simpleTableContainer.less]: done !

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**